src/devices/sound/lynx.h

```diff
@@ -13,36 +13,47 @@ class lynx_sound_device : public device_t, public device_sound_interface
 
 	lynx_sound_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
 
-	uint8_t read(offs_t offset);
-	void write(offs_t offset, uint8_t data);
+	u8 read(offs_t offset);
+	void write(offs_t offset, u8 data);
 	void count_down(int nr);
 	template <typename... T> void set_timer_delegate(T &&... args) { m_timer_delegate.set(std::forward<T>(args)...); }
 
 protected:
 	struct LYNX_AUDIO {
 		struct {
-			int8_t volume;
-			uint8_t feedback;
-			int8_t output;
-			uint8_t shifter;
-			uint8_t bakup;
-			uint8_t control1;
-			uint8_t counter;
-			uint8_t control2;
+			//const inline bool reset_done()     { return BIT(control1, 6); }       // Reset timer done flag (not implemented)
+			const inline bool   integrate_mode() { return BIT(control1, 5); }       // Integrate mode
+			const inline bool   reload_en()      { return BIT(control1, 4); }       // Reload enable
+			const inline bool   count_en()       { return BIT(control1, 3); }       // Count enable
+			const inline u8     timer_clock()    { return BIT(control1, 0, 3); }    // Timer clock
+			const inline bool   linked()         { return timer_clock() == 0b111; } // Linked timer?
+
+			//const inline bool last_clock()     { return BIT(control2, 2); }       // Last clock (not implemented)
+			//const inline bool borrow_in()      { return BIT(control2, 1); }       // Borrow in (not implemented)
+			//const inline bool borrow_out()     { return BIT(control2, 0); }       // Borrow out (not implemented)
```

Comment:
There’s no point using the `inline` keyword here – `inline` is implied when the body is placed in line with the declaration like this.

You’re not using the `const` qualifier correctly.  A `const` qualifier on a value return type has no effect.  What you mean to do is `const` qualify the member function so it can be called with a `const` instance (e.g. `bool integrate_mode() const { ... }`).